packages/transmute-cli/src/index.js

```diff
@@ -171,6 +170,6 @@ vorpal
  });
 
 vorpal
-  .delimiter('T$')
+  .delimiter('✨  $')
```

Comment:
This might have issues rendering on different kinds of command-lines.

Reply:
http://unicode.org/emoji/ its unicode.

Reply:
so not a problem, that I am aware of.